### ISSUE TITLE
[Feature Request] Undercut by multiples of a number

### ISSUE THREAD

**Issue by rikai** [tesu/PennyPincher#20]
Hello, I have a preference for undercutting by multiples of five, Ie, multiples of 5 or so.

Would it be possible to implement an option to undercut by certain multiples or something of the sort?

**Comment by zachwhite** [comment]
The new Modulo feature would theoretically allow for this, but the math is a little off.

Currently it's using this:
```
var price = listing.ItemListings[i].PricePerUnit - (listing.ItemListings[i].PricePerUnit % configuration.mod) - configuration.delta;
```

An additional step to apply the modulo adjustment _after_ the delta subtraction I think would fix it. Eg:
```
var price = listing.ItemListings[i].PricePerUnit - configuration.delta;
price = price - (price % configuration.mod);
```

A temporary workaround would be to set Modulo to 5, and Amount to Undercut By to 0. However, if the lowest price is already a multiple of 5, then you wouldn't actually undercut.

**Comment by tesu** [comment]
Fixed by #31

**Comment by rikai** [comment]
Thank you so much for this. :)
